app/views/article/_index_default.html.erb

```diff
@@ -18,8 +18,10 @@
     <li><%= doc_presenter.field_value('eds_abstract') %></li>
   </ul>
 <% end %>
-<% if document['eds_fulltext_links'].present? && !article_restricted?(document) %>
+<% if document.access_panels.online? && !article_restricted?(document) %>
   <ul class='document-metadata dl-horizontal dl-invert'>
-    <%= safe_join(doc_presenter.fulltext_links) %>
+    <% document.access_panels.online.links.each do |link| %>
+      <li><%= link_to(link.text, link.href) %></li>
```

Comment:
this change removes the class `article-fulltext-link` which is ok, but we can delete it from the CSS then.